### PR TITLE
[ART-3423] Lock 4.7-4.10 to RHEL 8.4 EUS content

### PR DESCRIPTION
**Pull request by yselkowitz**
4.6 and earlier are already locked to 8.2 EUS.

/cc @sosiouxme

### CODE REVIEW

File: core-services/release-controller/_repos/ocp-4.10-rhel8.repo [hunk 1]
```diff
@@ -59,7 +59,7 @@ gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release ht
 
 [rhel-8-nfv]
 name = rhel-8-nfv
-baseurl = https://cdn.redhat.com/content/dist/rhel8/8/x86_64/nfv/os/
+baseurl = https://cdn.redhat.com/content/tus/rhel8/8.4/x86_64/nfv/os/
```
[the remark this answers is not in the document]
Comment (yselkowitz) [reply core-services/release-controller/_repos/ocp-4.10-rhel8.repo:62]: Nope, RT and NFV are only available as TUS.

File: core-services/release-controller/_repos/ocp-4.7-rhel8.repo [hunk 2]
```diff
@@ -45,7 +45,7 @@ gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release ht
 
 [rhel-8-nfv]
 name = rhel-8-nfv
-baseurl = https://cdn.redhat.com/content/dist/rhel8/8/x86_64/nfv/os/
+baseurl = https://cdn.redhat.com/content/tus/rhel8/8.4/x86_64/nfv/os/
```
[the remark this answers is not in the document]
Comment (yselkowitz) [reply core-services/release-controller/_repos/ocp-4.7-rhel8.repo:48]: Same answer as above. :-)

File: core-services/release-controller/_repos/ocp-4.8-rhel8.repo [hunk 3]
```diff
@@ -59,7 +59,7 @@ gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release ht
 
 [rhel-8-nfv]
 name = rhel-8-nfv
-baseurl = https://cdn.redhat.com/content/dist/rhel8/8/x86_64/nfv/os/
+baseurl = https://cdn.redhat.com/content/tus/rhel8/8.4/x86_64/nfv/os/
```
[the remark this answers is not in the document]
Comment (yselkowitz) [reply core-services/release-controller/_repos/ocp-4.8-rhel8.repo:62]: Ditto

File: core-services/release-controller/_repos/ocp-4.9-rhel8.repo [hunk 4]
```diff
@@ -59,7 +59,7 @@ gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release ht
 
 [rhel-8-nfv]
 name = rhel-8-nfv
-baseurl = https://cdn.redhat.com/content/dist/rhel8/8/x86_64/nfv/os/
+baseurl = https://cdn.redhat.com/content/tus/rhel8/8.4/x86_64/nfv/os/
```
[the remark this answers is not in the document]
Comment (yselkowitz) [reply core-services/release-controller/_repos/ocp-4.9-rhel8.repo:62]: Ditto

**Comment by yselkowitz** [comment]
This or something similar needs to be merged ASAP to avoid RHEL 8.5 content from being used in CI (which would be out of sync with prod builds which are now locked into 8.4 EUS).

**Comment by jupierce** [comment]
/approve
/lgtm
Thanks @yselkowitz !

**Pull request merged**